Pages/Login/login.html

```diff
@@ -0,0 +1,44 @@
+<div class="MainLoginContainer" ng-controller="loginctrl">
```

Comment:
remove ng-controller="loginctrl" from here. Add it to routing configuration file.